src/woo_publications/publications/api/filters.py

```diff
@@ -0,0 +1,20 @@
+from django.utils.translation import gettext_lazy as _
+
+from django_filters.rest_framework import FilterSet, filters
+
+from ..models import Publication
+
+
+class PublicationFilterSet(FilterSet):
+    sorteer = filters.OrderingFilter(
```

Comment:
currently the fields are not camilized.

Reply:
okay - for these fields it doesn't matter yet so we can postpone it, the enum values are also not camelized which might be more odd-looking, but accepting it as is for now and we can figure out a solution if the UI team makes a remark about it